components/BookshelfItem.vue

```diff
@@ -161,6 +207,40 @@ function openContentURL(contentURL: ContentURL) {
   })
 }
 
+async function downloadUrl({ url, name, type }: { url: string, name: string, type: string }) {
+  try {
+    fileDownloading.value = true
```

Comment:
Boolean start with `is`/`can`/`should`/`must`
```suggestion
    isFileDownloading.value = true
```

Reply:
removed